kong/router/atc.lua

```diff
@@ -344,6 +344,16 @@ local function set_upstream_uri(req_uri, match_t)
 end
 
 
+-- captures has the form { [0] = full_path, [1] = capture1, [2] = capture2, ... }
+-- and captures[0] will be the full matched path
+-- this function tests if there are captures other than the full path
+-- by checking if there are 2 or more than 2 keys
+local function has_capture(captures)
+  local next_i = next(captures)
```

Comment:
`captures` should be possible to nil? so might need to check if it's nil before using the `next` function.